### PR TITLE
Migrate cfnet to ReLax

### PR DESCRIPTION
**Pull request by Grahams-Uncle**

### CODE REVIEW

**Comment by Grahams-Uncle** [comment]
This fixes #82

**Review by BirkhoffG — CHANGES_REQUESTED** [review]
Change every `RELax` to `ReLax`.

File: nbs/nbdev.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ project:
 
 website:
   title: "ReLax"
-  site-url: "https://birkhoffg.github.io/cfnet/"
+  site-url: "https://birkhoffg.github.io/relax/"
```
Comment (BirkhoffG) [inline nbs/nbdev.yml:6]: This should be https://birkhoffg.github.io/ReLax/

File: settings.ini [hunk 2]
```diff
@@ -21,8 +21,8 @@ nbs_path = nbs
 doc_path = _docs
 recursive = True
 doc_host = https://birkhoffg.github.io
-doc_baseurl = /cfnet/
-git_url = https://github.com/birkhoffg/cfnet/tree/master/
+doc_baseurl = /relax/
```
Comment (BirkhoffG) [inline settings.ini:24]: This should be `/ReLax/`

**Review by BirkhoffG — APPROVED** [review]

**Pull request merged**